packages/js-sdk/src/Framework.js

```diff
@@ -169,6 +189,26 @@ module.exports = class Framework {
         );
         return this.contracts.ISuperToken.at(wrapperAddress);
     }
+    /**
+     * @dev call to add a tx in the gas report. Does nothing if gas report type is not set.
+     * @param {tx oject} tx as returned by truffleContract action
+     * @param {str} actionName action title for row in report
+     */
+    _pushTxForGasReport(tx, actionName) {
+        this._gasMetering ? this._gasMetering.pushTx(tx, actionName) : null;
```

Comment:
I like this check pattern, which makes it much cleaner when adding the function  :+1: